node/blockchain/tests/test_rest_api/test_block.py

```diff
@@ -65,14 +82,33 @@ def test_blocks_pagination(api_client):
     assert len(results) == 1
     assert results[0] == json.loads(Block.objects.get(_id=1).body)
 
-    response = api_client.get('/api/blocks/?limit=1&offset=6')
+    response = api_client.get('/api/blocks/?limit=1&offset=7')
     assert response.status_code == 200
     response_json = response.json()
     results = response_json.get('results')
     assert isinstance(results, list)
     assert len(results) == 0
 
 
+@pytest.mark.parametrize(
+    'type_, example_name', (
+        (Type.GENESIS, BLOCK_DETAILS_GENESIS_EXAMPLE),
```

Comment:
we can just use Type enum as a list for all type. This way we will not forget to add them here when new types are created